### PR TITLE
GLT-2540: added Other MISO Tools section to user manual

### PR DESCRIPTION
**Pull request by djcooke**

### CODE REVIEW

File: docs/user_manual/other_miso_tools.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+{% include userman-toplink.md %}
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Identity Search" %}
+
+Note: This tool only appears if your site uses [detailed sample mode](site_configuration.html#detailed_sample_mode).
+
+The Identity Search tool can be used to find identity samples with a specified external name, and to see all of the
```
Comment (callunity) [inline docs/user_manual/other_miso_tools.md]: maybe `identity (donor) samples` , as you use the term 'donor' later in the paragraph

File: docs/user_manual/other_miso_tools.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+children of the resulting identities. This is useful if you're not sure whether you've received samples from this donor
```
Comment (callunity) [inline docs/user_manual/other_miso_tools.md]: I worry that `children of the resulting identities` may be ambiguous, but I'm not sure what would be better... `samples associated with the resulting identities`?

File: docs/user_manual/other_miso_tools.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+The following tools can help you with planning or finding things within MISO. They can be found by clicking the
+appropriate link in the Tools list in the menu on the left side of the screen.
+
+{% assign sub = 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Index Distance" %}
+
+When you run multiple libraries in a sequencing lane or partition, known as multiplexing, indices are used to
+identify the individual libraries and demultiplex (separate) the data afterwards. If two or more libraries in the pool
+have identical or near-identical barcodes, demultiplexing may be more difficult or even impossible. The Index Distance
```
[the remark this answers is not in the document]
Comment (djcooke) [reply docs/user_manual/other_miso_tools.md]: fixed

File: docs/user_manual/other_miso_tools.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+tool can help you identify these clashing indices within a set.
```
[the remark this answers is not in the document]
Comment (djcooke) [reply docs/user_manual/other_miso_tools.md]: changed to "indices which may be inadvisable to multiplex together"

File: docs/user_manual/other_miso_tools.md (hunk 1 above)
Comment (djcooke) [reply docs/user_manual/other_miso_tools.md]: changed to "identity (donor/organism) samples"

File: docs/user_manual/other_miso_tools.md (hunk 2 above)
Comment (djcooke) [reply docs/user_manual/other_miso_tools.md]: changed to "samples descended from the resulting identities" as I think that makes the relationship a bit clearer

**Pull request merged**